analyses/README.md

```diff
@@ -52,7 +52,7 @@ Note that _nearly all_ modules use the harmonized clinical data file (`pbta-hist
 | [`sex-prediction-from-RNASeq`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/sex-prediction-from-RNASeq) | `pbta-gene-expression-kallisto.stranded.rds` <br> `pbta-histologies.tsv` | Predicts genetic sex using RNA-seq data ([#84](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/84)) | N/A | No |
 | [`snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers) | `pbta-snv-lancet.vep.maf.gz` <br> `pbta-snv-mutect2.vep.maf.gz` <br> `pbta-snv-strelka2.vep.maf.gz` <br> `pbta-snv-vardict.vep.maf.gz` <br> `tcga-snv-lancet.vep.maf.gz` <br> `tcga-snv-mutect2.vep.maf.gz` <br> `tcga-snv-strelka2.vep.maf.gz` | Generates consensus SNV and indel calls for PBTA and TCGA data; calculates tumor mutation burden using the consensus calls | `results/consensus/pbta-snv-consensus-mutation.maf.tsv.gz` (included in data download; too large for tracking via GitHub) <br> `results/consensus/pbta-snv-consensus-mutation-tmb-all.tsv` <br> `results/consensus/pbta-snv-consensus-mutation-tmb-coding.tsv`(included in data download; too large for tracking via GitHub) <br> `results/consensus/tcga-snv-consensus-mutation.maf.tsv.gz` <br> `results/consensus/tcga-snv-mutation-tmb.tsv` <br> `results/consensus/tcga-snv-mutation-tmb-coding.tsv` | Yes |
 | [`ssgsea-hallmark`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/ssgsea-hallmark) | `pbta-gene-counts-rsem-expected_count.stranded.rds` | *Deprecated*; performs GSVA using Hallmark gene sets | N/A | Yes |
-| [`survival-analysis`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/survival-analysis) | TBD | *In progress*. Provides survival analysis template and performs several different survival analyses | N/A | Yes |
+| [`survival-analysis`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/survival-analysis) | `pbta-histologies.tsv` <br> `independent-specimens.wgswxs.primary.tsv` (results from independent-samples module) <br> `TelomeraseScores_PTBAStranded_FPKM.txt` (results from telomerase-activity-prediction module) <br> `tp53_altered_status.tsv` (from results by tp53_nf1_score module) <br> `quantiseq_deconv-output.rds` (results from immune-deconv module) <br> `pbta-gene-expression-rsem-fpkm-collapsed.polya.rds` and `pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds` (results from collapse-rnaseq module) | `cox_reg_results_per_{covariates}.tsv` (see module results for details) | N/A
```

Comment:
For the `Input Files` column,
+ Add backticks around module names
+ Don't refer to a module unless the data is literally read in from that module. Some of these, eg. `independent-specimens.wgswxs.primary.tsv` are in the data release, and their origin is documented elsewhere. 

+ The third column `Brief Description` should contain a description of the module, but it right now shows a TSV file name. I think you meant to put that file name in the next column `Output Files Consumed by Other Analyses`, but actually I'd leave this as `N/A` since none of the results files are actually consumed by another module. They are used to make figures, but not to perform analyses elsewhere.

+ Fill in the last column with "Yes"